var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -102,22 +138,8 @@ def configure_args(self):
                 args.append("--disable-tiff")
 
             if "+curl" in self.spec:
-                args.append("--with-curl=" + self.spec["curl"].prefix.bin.join("curl-config"))
+                args.append("--with-curl=" + self.pkg.spec["curl"].prefix.bin.join("curl-config"))
```

Comment:
Does `self.spec` not work? We're still using that other places in the builder

Reply:
Which attributes can be referenced from the builders vs which need to be pipped through the package is very unclear, and the builder parent classes refer to the spec by `self.pkg.spec` so I erred on the side of "doing what is being done already". I don't actually know if it does or not.

Reply:
Ping @alalazo...

Reply:
Tried it out myself, `self.spec` works from the builder.